[CORE] add minus, minusEquals methods to FixedOps*

nknize opened the pull request:
Proposed PR to add matrix subtraction support to FixedOps. Includes unit tests

lessthanoptimal commented:
Thanks for the pull request.  Could you make a modification to GenerateFixedMatrixNxN and GenerateFixedOps which create those changes?  The comment below also needs to be added to FixedMatrix*x*

<p>DO NOT MODIFY.  Automatically generated code created by GenerateFixedMatrixNxN</p>

I know the auto code generators can be ugly at times.  I can also do the changes myself.

nknize commented:
Updated GenerateFixedOps.java to auto generate minus and minusEquals, and GenerateFixedMatrixNxN.java to auto generate overridden equals method.  Also added DO NOT MODIFY tag to GenerateFixedMatrixNxN.

lessthanoptimal commented:
I've basically accepted most of this pull request.  It couldn't be auto merged and a few things needed to be done differently so I ended up incorporating the changes manually and acknowledging you in the readme.  Thanks for all of your pull requests!
